app/models/planet.py

```diff
@@ -0,0 +1,23 @@
+from app import db
```

Comment:
Just noticed you have an `app/model/planet.py` and `app/models/planet.py`  So.... one isn't needed.